examples/src/main/scala/zio/dynamodb/examples/LocalDdbServer.scala

```diff
@@ -0,0 +1,27 @@
+package zio.dynamodb.examples
+
+import com.amazonaws.services.dynamodbv2.local.main.ServerRunner
+import com.amazonaws.services.dynamodbv2.local.server.DynamoDBProxyServer
+import zio.blocking.{ effectBlocking, Blocking }
+import zio.{ Has, ZLayer, ZManaged }
+
+// TODO: move to production code
```

Comment:
For this todo do you just mean having this in-memory version as an option for consumers of zio-dynamodb to point to for testing?

Reply:
ah yes - I was think about moving this to production code now that it is used in multiple places. However if users want to use they will also have to make changes to their SBT build files so its not the cleanest of components. I will remove the comment for now and lets see what feedback we get.